alf/algorithms/actor_critic_algorithm.py

```diff
@@ -170,8 +176,6 @@ def calc_loss(self, training_info, final_time_step, final_policy_step):
         ac_loss = self._loss(training_info, training_info.info.value,
                              final_time_step, final_value)
 
-        self.add_reward_summary("reward", training_info.reward)
-
```

Comment:
We should still keep this one, this is reward for training.
I am also thinking that we probably need to document the meaning of `reward` clearly, otherwise it might be used incorrectly, Or perhaps we should use different name to differentiate different reward: 'environment_reward' and 'reward`

Reply:
It's not gone. It was moved to the parent class on_policy_algorithm.py with the appropriate name